test/integration/default/inspec.yml

```diff
@@ -14,8 +14,8 @@
 
 name: cloud-pubsub
 title: Google Cloud Pub/Sub
-version: 0.1.0
-inspec_version: '~> 2.3.24'
+version: 0.2.1
+inspec_version: '~> 3.9.3'
```

Comment:
This attribute should be dropped because Kitchen-Terraform manages the version of InSpec.